Jakefile

```diff
@@ -161,17 +161,30 @@ function generateChromeManifest() {
   });
 }
 
+/**
+ * Exports the files to a target, used to export into mozilla-central
+ *
+ */
+function exportFiles(target) {
+  let mcLocation = getMozillaCentralLocation();
+  let extTargetDir = path.join(mcLocation, target);
+  jake.rmRf(extTargetDir);
+  jake.cpR(BUILD_DIR, extTargetDir);
+
+  console.log(`Exported built sources into ${extTargetDir}`);
+}
+
 desc(`Builds the extension into the ${BUILD_DIR}/ directory`);
 task("build", ["building:cleanup", "building:copy"], () => {});
 
 desc("Exports the sources into mozilla-central");
 task("export-mc", ["build"], () => {
-  let mcLocation = getMozillaCentralLocation();
-  let extTargetDir = path.join(mcLocation, "browser/extensions/webcompat");
-  jake.rmRf(extTargetDir);
-  jake.cpR(BUILD_DIR, extTargetDir);
+  exportFiles("browser/extensions/webcompat");
+});
 
-  console.log(`Exported built sources into ${extTargetDir}`);
+desc("Exports the sources into the mozilla-central for android");
+task("export-mc-android", ["build"], () => {
```

Comment:
I am very sure that people who are going to review our updates to the desktop tree are not willing/able to review the mobile changes as well, so landing changes for fennec and desktop has to happen in two bugs, with two commits and two sets of reviewers. That's the reason I wrote a second exporter.

> But I don't know if other reviewer will have concern to put same source code in 2 different folders.

I talked to some people and so far, it seems like nobody has an issue with that since there is no other possibility to share sources. however, we are the first people to ever do that, so it may be that there will be additional discussion on landing. If it turns out we need to do something else, we can do that, but for now, this seems good enough.